### PR TITLE
Add initial RDF jsonschema

### PR DESCRIPTION
**Pull request by constantinpape**
Based on https://bioimage.io/docs/#/contribute_models/resource-description-file

TODO
- [ ] Add more validation, e.g. for urls and version strings
- [ ] Clarify if `version` refers to the version of this resource or the bioimage spec version

### CODE REVIEW

**Comment by FynnBe** [comment]
We would also be able to generate this by adding another schema class `schema.RDF(schema.Spec)` that inherits from [schema.Spec](https://github.com/bioimage-io/python-bioimage-io/blob/c4fdc845a3b4f11af078bc1df6398e538b9c4ad9/pybio/spec/schema.py#L42)

Some fields currently in `schma.Spec` might have to be moved to `schema.Model(schema.Spec)`. The idea was to have `schema.Spec` as the minimal requirements schema for any specification. Some fields like `badges` should maybe not be valid for Model, but there is a big overlap...

**Comment by constantinpape** [comment]
> We would also be able to generate this by adding another schema class `schema.RDF(schema.Spec)` that inherits from [schema.Spec](https://github.com/bioimage-io/python-bioimage-io/blob/c4fdc845a3b4f11af078bc1df6398e538b9c4ad9/pybio/spec/schema.py#L42)

I think we decided here to go with jsonschema directly in order to not complicate things.

**Comment by oeway** [comment]
@constantinpape This is great! Thanks!

**Comment by constantinpape** [comment]
Thanks @oeway. I will try to add some more validation tomorrow.

File: rdf.schema.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        "git_repo": {
+            "type": "string",
+            "description": "url to the git repo"
+        },
+        "license": {
+            "type": "string",
+            "description": "the license name for the resource, e.g. `MIT`"
+        },
+        "tags": {
+            "type": "array",
```
Comment (oeway) [inline rdf.schema.json:37]: Maybe also add for all the `array` types.
```
"items": {
              "type": "string"
            },
```

**Comment by constantinpape** [comment]
@oeway I have added the items for all arrays now.
There is some more validation that could be done using regex, but I think it's maybe better to merge this now and add this gradually.

**Comment by oeway** [comment]
Nice! I did a minor modification on the description.

[Rendered web form](https://rjsf-team.github.io/react-jsonschema-form/#eyJmb3JtRGF0YSI6e30sInNjaGVtYSI6eyIkc2NoZW1hIjoiaHR0cDovL2pzb24tc2NoZW1hLm9yZy9kcmFmdC0wNy9zY2hlbWEiLCIkaWQiOiJodHRwczovL3Jhdy5naXRodWJ1c2VyY29udGVudC5jb20vYmlvaW1hZ2UtaW8vc3BlYy1iaW9pbWFnZS1pby9tYXN0ZXIvcmRmLnNjaGVtYS5qc29uIiwidGl0bGUiOiJCaW9JbWFnZS5JTyBSZXNvdXJjZSBEZXNjcmlwdGlvbiBGaWxlIEZvcm1hdCBTY2hlbWEiLCJkZXNjcmlwdGlvbiI6IlJlc291cmNlIERlc2NyaXB0aW9uIEZpbGUgU2NoZW1hIiwicHJvcGVydGllcyI6eyJpZCI6eyJ0eXBlIjoic3RyaW5nIiwiZGVzY3JpcHRpb24iOiJ1bmlxdWUgaWRlbnRpZmllciBmb3IgdGhlIHJlc291cmNlLCB5b3UgY2FuIGFsc28gdXNlIGEgYGRvaWAgZS5nLiBnZW5lcmF0ZWQgZnJvbSBbemVub2RvXShodHRwczovL3plbm9kby5vcmcvKSJ9LCJ0eXBlIjp7InR5cGUiOiJzdHJpbmciLCJkZXNjcmlwdGlvbiI6InR5cGUgb2YgdGhlIHJlc291cmNlLCBlLmcuIGBtb2RlbGAsIGBkYXRhc2V0YCwgYGFwcGxpY2F0aW9uYCBvciBgbm90ZWJvb2tgIiwiZW51bSI6WyJtb2RlbCIsImRhdGFzZXQiLCJhcHBsaWNhdGlvbiIsIm5vdGVib29rIl19LCJuYW1lIjp7InR5cGUiOiJzdHJpbmciLCJkZXNjcmlwdGlvbiI6Im5hbWUgb2YgdGhlIHJlc291cmNlLCBhIGh1bWFuLWZyaWVuZGx5IG5hbWUifSwic291cmNlIjp7InR5cGUiOiJzdHJpbmciLCJkZXNjcmlwdGlvbiI6InVybCB0byB0aGUgc291cmNlIG9mIHRoZSByZXNvdXJjZSJ9LCJkb3dubG9hZF91cmwiOnsidHlwZSI6InN0cmluZyIsImRlc2NyaXB0aW9uIjoidXJsIHRvIHJoZSB6aXBwZWQgZmlsZSBpZiBhcHBsaWNhYmxlIn0sImdpdF9yZXBvIjp7InR5cGUiOiJzdHJpbmciLCJkZXNjcmlwdGlvbiI6InVybCB0byB0aGUgZ2l0IHJlcG8ifSwibGljZW5zZSI6eyJ0eXBlIjoic3RyaW5nIiwiZGVzY3JpcHRpb24iOiJ0aGUgbGljZW5zZSBuYW1lIGZvciB0aGUgcmVzb3VyY2UsIGUuZy4gYE1JVGAifSwidGFncyI6eyJ0eXBlIjoiYXJyYXkiLCJkZXNjcmlwdGlvbiI6ImEgbGlzdCBvZiB0YWdzIiwiaXRlbXMiOnsidHlwZSI6InN0cmluZyJ9fSwiYXR0YWNoZW1lbnRzIjp7InR5cGUiOiJvYmplY3QiLCJkZXNjcmlwdGlvbiI6ImEgZ3JvdXAgb2YgYXR0YWNobWVudHMsIGxpc3Qgb2YgcmVzb3VyY2VzIGdyb3VwZWQgYnkga2V5In0sImF1dGhvcnMiOnsidHlwZSI6ImFycmF5IiwiZGVzY3JpcHRpb24iOiJ0aGUgZnVsbCBsaXN0IG9mIGF1dGhvciBuYW1lcyIsIml0ZW1zIjp7InR5cGUiOiJzdHJpbmcifX0sImJhZGdlcyI6eyJ0eXBlIjoiYXJyYXkiLCJkZXNjcmlwdGlvbiI6ImEgbGlzdCBvZiBiYWRnZXMiLCJpdGVtcyI6eyJ0eXBlIjoic3RyaW5nIn19LCJjaXRlIjp7InR5cGUiOiJhcnJheSIsImRlc2NyaXB0aW9uIjoiaG93IHRvIGNpdGUgdGhlIHJlc291cmNlIiwiaXRlbXMiOnsidHlwZSI6InN0cmluZyJ9fSwiZG9jdW1lbnRhdGlvbiI6eyJ0eXBlIjoic3RyaW5nIiwiZGVzY3JpcHRpb24iOiJ1cmwgdG8gdGhlIGRvY3VtZW50YXRpb24gaW4gbWFya2Rvd24gZm9ybWF0In0sImljb24iOnsidHlwZSI6InN0cmluZyIsImRlc2NyaXB0aW9uIjoiYW4gaWNvbiBmb3IgdGhlIHJlc291cmNlIn0sImxpbmtzIjp7InR5cGUiOiJhcnJheSIsImRlc2NyaXB0aW9uIjoiYSBsaXN0IG9mIGxpbmtlZCByZXNvdXJjZXMsIGFuIGlkIHRvIG90aGVyIHJlc291b3JjZXMiLCJpdGVtcyI6eyJ0eXBlIjoic3RyaW5nIn19LCJ2ZXJzaW9uIjp7InR5cGUiOiJzdHJpbmciLCJkZXNjcmlwdGlvbiI6InRoZSB2ZXJzaW9uIG51bWJlciBmb3IgdGhlIHJlc291cmNlLCBzdGFydGluZyBmcm9tIGAwLjEuMGAifX0sInJlcXVpcmVkIjpbImlkIiwidHlwZSIsIm5hbWUiLCJkZXNjcmlwdGlvbiJdLCJhZGRpdGlvbmFsUHJvcGVydGllcyI6dHJ1ZX0sInVpU2NoZW1hIjp7fSwidGhlbWUiOiJtYXRlcmlhbC11aSIsImxpdmVTZXR0aW5ncyI6eyJ2YWxpZGF0ZSI6ZmFsc2UsImRpc2FibGUiOmZhbHNlLCJvbWl0RXh0cmFEYXRhIjpmYWxzZSwibGl2ZU9taXQiOmZhbHNlfX0=)

**Pull request merged**